docs/1-Applying/resumes.md

```diff
@@ -74,9 +74,9 @@ I'd recommend checking out other people's resumes too, as some of them have sect
 
 As much as possible, don't list your skills without context proving you have them. Anyone can write "good communicator" on their resume, and unless you talk about something you did that needed very good communication skills, that line is essentially useless. In general, just don't list "good communicator" as a skill and instead talk about your communication skills when you talk about your experience. This is also true for a lot of soft skills such as "effective team member" or "strong personal leadership".
 
-Similarily, show that you know programming languages by mentioning them in the work you've done with them and don't just list a bunch of them at the top of your resume. It's useful to have them all in one place of course. But as a general rule of thumb, any language/technology/acronym that you put in a list should also appear elsewhere on your resume.
+Similarily, show that you know programming languages by mentioning them in the work you've done with them and don't just list a bunch of them at the top of your resume. It's useful to have them all in one place of course. But ideally, languages/technologies/acronyms that you put in a list should also appear elsewhere on your resume.
 
-(There are some exceptions to this. For example, you might want to include Haskell to show that you like to challenge yourself intellectually, but your largest Haskell project is smaller than all your other projects. It's fine to include Haskell, as long as you are able to explain how you've used Haskell during an interview if asked. In general, you should be able to talk about anything on your resume during an interview).
+We say ideally because you might have dabbled in some new technology just to try it (e.g. some Javascript framework, a functional language, some API, etc). That still signals to the employers that you can learn on your own which is great [see side projects](side_projects.md). It's fine to include them, as long as you are able to explain how you've used them during an interview if asked. In general, you should be able to talk about anything on your resume during an interview.
```

Comment:
also this example is confusing to me - if you do a side project, often that'll be on your resume